pyproject.toml

```
@@ -3,9 +3,10 @@ name = "flowcean"
 version = "0.3.0"
 description = "Automatic generation of models for cyber-physical systems."
 readme = "README.md"
-requires-python = "~=3.12.7"
+requires-python = ">=3.10,<3.13"
```

Following principles of semantic versioning, we should use
```suggestion
requires-python = ">=3.10,<4.0"
```